### ISSUE TITLE
[HELP] Can't establish communication : 503

### ISSUE THREAD

**Issue by MatiasRizzi** [OCSInventory-NG/OCSInventory-ocsreports#738]
_Dear all_

_I'm setting up the first ocsinventory and I have this problem in the server. Since I probe with different agents and produce same logs in apache._

**Server installation method** 

- [ ] DEB Package
- [ ]  RPM Package
- [ x ] Install SH
- [ ] Manual installation

**Server informations**

- SO Debian Strech
- PHP 7.0.33
- MySql  Version 15.1 Distrib 10.1.38-MariaDB,
- Apache version : 2.4.25
- Ocs server version : 2.5

**Agent Informations**
- SO Centos 7
- Ocs agent version : 2.4.1

**Error log Apache:**
```
[:error] [pid 16195] [client IP:49206] PHP Warning:  vsprintf(): Too few arguments in /usr/share/ocsinventory-reports/ocsreports/require/function_commun.php on line 74, referer: http://[IPServer]/ocsreports/index.php?function=visu_computers
[:error] [pid 16195] [client IP:49206] PHP Warning:  mysqli_query(): Empty query in /usr/share/ocsinventory-reports/ocsreports/require/function_commun.php on line 103, referer: http://[IPServer]/ocsreports/index.php?function=visu_computers
```
**Error log Agent**
```
[debug] Ocsinventory unified agent for UNIX, Linux and MacOSX 2.4.2
[debug] Log system initialised (File)
[debug] --scan-homedirs missing. Don't scan user directories
[debug] accountconfig file: `/var/lib/ocsinventory-agent/http:__[IPServer]_ocsreports/ocsinv.conf doesn't exist. I create an empty one
[debug] ocsinv.conf updated successfully
[debug] ocsinv.conf updated successfully
[debug] Accountinfo file: /var/lib/ocsinventory-agent/http:__[IPServer]_ocsreports/ocsinv.adm
[info] Accountinfo file doesn't exist. I create an empty one.
[debug] Invalid parameter while writing accountinfo file
[debug] Account info updated successfully
[debug] OCS Agent initialised
[debug] Turns hooks on for /etc/ocsinventory/modules.conf
[debug] Ocsinventory unified agent for UNIX, Linux and MacOSX 2.4.2
 Log system initialised (File)
[debug] Ocsinventory unified agent for UNIX, Linux and MacOSX 2.4.2
[debug] Log system initialised (File)
[debug] Calling handlers : `start_handler'
[debug] [snmp] Calling snmp_start_handler
[debug]  - Net::SNMP loaded
[debug] [download] Calling download_start_handler
[debug]  - LWP loaded
[debug]  - LWP::Protocol::https loaded
[debug] Compress::Zlib is available.
[debug] Calling handlers : `prolog_writer'
[debug] sending XML
[debug] sending: <?xml version="1.0" encoding="UTF-8"?>
<REQUEST>
  <DEVICEID>pruebasc7-2019-05-13-12-18-43</DEVICEID>
  <QUERY>PROLOG</QUERY>
</REQUEST>
[error] Cannot establish communication : 503 Service Unavailable
```

_I already try all the alternatives that exist in wiki, inlcusive installing the same versions between server and agent_

**Comment by MatiasRizzi** [comment]
> 
> **Error log Apache:**
> 
> ```
> [:error] [pid 16195] [client IP:49206] PHP Warning:  vsprintf(): Too few arguments in /usr/share/ocsinventory-reports/ocsreports/require/function_commun.php on line 74, referer: http://[IPServer]/ocsreports/index.php?function=visu_computers
> [:error] [pid 16195] [client IP:49206] PHP Warning:  mysqli_query(): Empty query in /usr/share/ocsinventory-reports/ocsreports/require/function_commun.php on line 103, referer: http://[IPServer]/ocsreports/index.php?function=visu_computers
> ```
Error in apache was solved updating server to 2.5. But Error 503 in agent log persist

**Comment by MatiasRizzi** [comment]
It's already working. Install with RPM following this guide: https://celerium.org/2018/11/03/setup-install-ocsng-on-centos-7/
It started to work when the agent sent a report to the server.
